truetype/truetype.go

```diff
@@ -95,11 +99,16 @@ type cm struct {
 	start, end, delta, offset uint32
 }
 
+// An nr holds a font's name, style
+type nr struct {
+	fontInfo [2]string
```

Comment:
fontInfo is not a Truetype concept.  Would `nameID` be better?

I chose `[2]string` because the values of interest, name and style, are nameIDs 1 and 2, respectively.  `[2]string` let me avoid an extra switch statement on line 354.